dts/arm/st/l0/stm32l051.dtsi

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2019 Linaro Ltd.
+ * Copyright (c) 2019 Centaur Analytics, Inc
+ * Copyright (c) 2021 Nomono AS
```

Comment:
You can keep only your copyright. Apply in each newly added file

Reply:
ok